### ISSUE TITLE
TALEdit crash android application

### ISSUE THREAD

**Issue by mateusvicente100** [Zeus64/alcinoe#143]
Hello. I don't know any group about the component, so, I decided to put my problem here.
I installed the alcinoe to use visual components. The samples about ALFmxControls works fine. In my project all works fine, but when I put an TALEdit I got an access violation just in android. 
I don't know what can be. If I add all my project units to ALFmxControls project I figured out, but if I just create an new application, put one TALEdit I got app crash in android device with access violation.

Has something special to do when we create an new project to use visual alcinoe components?

**Comment by viniciusfbb** [comment]
Mateus, qual a versão do seu delphi?

**Comment by mateusvicente100** [comment]
Hello. Sorry for my late answer, but I tested just now, because I was doing other things here in the company. @GaNacereddine thanks so much guy, I didn't read anything about the JAR files. When I added it to my project all works greats 👍 

@viniciusfbb Obrigado também pela ajuda. Eu dei um review na ordem dos path conforme você sugeriu. Estava de acordo. Acredito que o problema era apenas os arquivos JAR.
